housekeeping/docker/kagome-dev/build_and_push.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -ex
+
+cd "$(dirname "$0")"
+
+VERSION=1
+TAG=harrm/kagome-dev:$VERSION
+
+docker build -t ${TAG}-ubuntu -f ubuntu.Dockerfile .
+docker push ${TAG}-ubuntu
```

Comment:
we can remove ubuntu, and clean CI.

I don't see reasons to keep using `ubuntu`. `minideb` and `ubuntu` are both Debian-based distros. 
you should be able to install all dependency to `minideb`.
With `alpine` we couldn't test our code against all compilers (`gcc*, clang*`), it is complicated to install compilers to alpine (you have compile them from source using default gcc). so we had ubuntu.

we use Alpine not because it lightweight (but yes it played the role). Alpine is based on `musl libc` which is considered to be more secure and faster.